resource/opentelemetry-resource-detector-azure/pyproject.toml

```diff
@@ -26,7 +26,7 @@ classifiers = [
 ]
 dependencies = [
   "opentelemetry-sdk ~= 1.21",
-  "opentelemetry-instrumentation == 0.47b0.dev",
+  "opentelemetry-instrumentation ~= 0.43b0",
```

Comment:
Any reason why `0.43b0` was chosen specifically?

Reply:
Because it's when suppress_instrumentation helper has been introduced